zanata-war/src/main/webapp/language/language.xhtml

```diff
@@ -140,6 +177,17 @@
         </div>
       </div>
     </div>
+    <s:fragment rendered="#{identity.loggedIn}">
+      <ui:include src="../WEB-INF/layout/language/contact_coordinator_modal.xhtml">
```

Comment:
This should not need to use `../WEB-INF`, should work with just `/WEB-INF` and will be more robust.